From 231be3d0c31500e6c993ff22e48afae84c0a333c Mon Sep 17 00:00:00 2001
From: Evgeny Lensky <surfernsk@gmail.com>
Date: Fri, 24 Nov 2017 01:50:45 +0700
Subject: [PATCH] Fix check for system libraries in both lib64 and lib

if host_cpu = [i?86] libsubdirs = "", but it is necessary libsubdirs = "lib"
Invalid execution, superfluous comma.
---
 m4/ax_boost_base.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/m4/ax_boost_base.m4 b/m4/ax_boost_base.m4
index 19c71b6..2bce519 100644
--- m4/ax_boost_base.m4
+++ m4/ax_boost_base.m4
@@ -33,7 +33,7 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 42
+#serial 43
 
 # example boost program (need to pass version)
 m4_define([_AX_BOOST_BASE_PROGRAM],
@@ -114,7 +114,7 @@ AC_DEFUN([_AX_BOOST_BASE_RUNDETECT],[
     AS_CASE([${host_cpu}],
       [x86_64],[libsubdirs="lib64 libx32 lib lib64"],
       [ppc64|s390x|sparc64|aarch64|ppc64le],[libsubdirs="lib64 lib lib64"],
-      [libsubdirs="lib"],
+      [libsubdirs="lib"]
     )
 
     dnl allow for real multi-arch paths e.g. /usr/lib/x86_64-linux-gnu. Give
-- 
2.17.1

